issue2pr.py

```diff
@@ -9,10 +9,27 @@
 
 from __future__ import print_function
 
+import argparse
+import getpass
+import json
```

Comment:
Oh yeah, I see what you're saying.  If you pass in a dict directly it will send it form-encoded, but yes the GH API just accepts all JSON data.